chia/_tests/wallet/test_new_wallet_protocol.py

```diff
@@ -359,32 +368,14 @@ async def test_request_coin_state_limit(one_node: OneNode, self_hostname: str) -
 
     response = wallet_protocol.RespondCoinState.from_bytes(resp.data)
 
-    assert set(response.coin_ids) == set(coin_records.keys())
-    assert len(response.coin_states) == len(coin_records) - 10000
+    assert response.coin_ids == list(coin_records.keys())[:100000]
+    assert len(response.coin_states) == len(coin_records) - 20000
 
     for coin_state in response.coin_states:
         coin_record = coin_records[coin_state.coin.name()]
         assert coin_record.coin_state == coin_state
         assert coin_record.confirmed_block_index > 1
 
-    # The expected behavior when the limit is exceeded, is to skip the rest
-    # The order is not guaranteed, so it can't be relied upon here
-    # Just request the remaining coin ids in subsequent request(s)
```

Comment:
did this behavior change? Do we just get an error now, and none of the coins were subscribed to?

Reply:
Yes